packages/client-core/src/components/Debug/index.tsx

```diff
@@ -25,6 +27,16 @@ export const Debug = () => {
           toggleAvatarDebug()
         }
       }
+
+      if (ev.key === 'q') {
+        if (document.activeElement?.querySelector('canvas')) {
```

Comment:
lets move this to a keybind in AvatarInputSchema and surround it with an `if (isDev)` check